packages/ui/src/modal/list/ListModalRenderer.tsx

```diff
@@ -108,9 +99,7 @@ const isCurrencyAllowed = (
       case 'reservoir':
         return true
       case 'opensea':
-        return openseaPaymentTokens.some(
-          (token) => token.address === currency.contract
-        )
+        return false
```

Comment:
Can we add a comment here to explain that we're planning to come back to fix this properly?